<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OknoEdytora</class>
 <widget class="QDialog" name="OknoEdytora">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>930</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OknoEdytora</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Producent</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="producent">
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Mobilny system operacyjny</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="system_operacyjny"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>System operacyjny</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_3"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Cena</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_4"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Waga</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_5"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Włączony?</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QPushButton" name="zapisz">
         <property name="text">
          <string>Zapisz</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="wczytaj">
         <property name="text">
          <string>Wczytaj</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="wlacz">
         <property name="text">
          <string>Włącz</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="wylacz">
         <property name="text">
          <string>Wyłącz</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Zainstalowane Aplikacje</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="lista_app"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLineEdit" name="kategoria_app">
           <property name="text">
            <string>Kategoria</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="nazwa_app">
           <property name="text">
            <string>Nazwa</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="dev_app">
           <property name="text">
            <string>Deweloper</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="pamiec_app">
           <property name="text">
            <string>Pamięć [B]</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="wersja_app">
           <property name="text">
            <string>Wersja</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QPushButton" name="dodaj_app">
           <property name="text">
            <string>Dodaj</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="usun_app">
           <property name="text">
            <string>Usuń</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>dodaj_app</sender>
   <signal>clicked()</signal>
   <receiver>OknoEdytora</receiver>
   <slot>dodajAplikacje()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>540</x>
     <y>574</y>
    </hint>
    <hint type="destinationlabel">
     <x>464</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>usun_app</sender>
   <signal>clicked()</signal>
   <receiver>OknoEdytora</receiver>
   <slot>usunAplikacje()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>793</x>
     <y>574</y>
    </hint>
    <hint type="destinationlabel">
     <x>464</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>zapisz</sender>
   <signal>clicked()</signal>
   <receiver>OknoEdytora</receiver>
   <slot>zapisz()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>496</y>
    </hint>
    <hint type="destinationlabel">
     <x>464</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>wczytaj</sender>
   <signal>clicked()</signal>
   <receiver>OknoEdytora</receiver>
   <slot>wczytaj()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>159</x>
     <y>496</y>
    </hint>
    <hint type="destinationlabel">
     <x>464</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>dodajAplikacje()</slot>
  <slot>usunAplikacje()</slot>
  <slot>zapisz()</slot>
  <slot>wczytaj()</slot>
 </slots>
</ui>
